script/submit-definitions

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Usage: script/update-nodes [-r [<remote>]]
+#
+# scrapes, commits, pushes, and pull-requests new node definitions
+#
+# * inspired by: https://github.com/jasonkarns/brew-publish *
+
+set -e
+
+while getopts ":r" opt; do
+  case $opt in
+  r) brew_remote="${OPTARG:-origin}" ;;
```

Comment:
Not particularly important, but this script doesn't have anything to do with `brew`. Should probably rename to `git_remote`